noir-projects/aztec-nr/aztec/src/unencrypted_logs/unencrypted_event_emission.nr

```diff
@@ -0,0 +1,58 @@
+use crate::{
+    context::{PrivateContext, PublicContext}, event::event_interface::EventInterface,
+    encrypted_logs::payload::compute_encrypted_event_log, oracle::logs_traits::LensForEncryptedEvent
+};
+use dep::protocol_types::{address::AztecAddress, grumpkin_point::GrumpkinPoint, traits::Serialize};
+
+fn emit_with_keys<Event, NB, MB, OB, N, M>(
```

Comment:
Oh, naming here should be altered. Have some other comments when I thought a bit more about it that likely makes sense here. Lets discuss it later.